tools/GenerateTestSummary/TestSummaryGenerator.cs

```diff
@@ -144,21 +144,17 @@ public static void CreateSingleTestSummaryReport(string trxFilePath, StringBuild
 
                 """);
 
-                var errorMsgBuilder = new StringBuilder();
-                errorMsgBuilder.AppendLine(test.Output?.ErrorInfo?.InnerText ?? string.Empty);
+                reportBuilder.AppendLine();
+                reportBuilder.AppendLine("```yml");
+
+                reportBuilder.AppendLine(test.Output?.ErrorInfo?.InnerText ?? string.Empty);
                 if (test.Output?.StdOut is not null)
                 {
-                    errorMsgBuilder.AppendLine();
-                    errorMsgBuilder.AppendLine("### StdOut");
-                    errorMsgBuilder.AppendLine(test.Output.StdOut);
+                    reportBuilder.AppendLine();
+                    reportBuilder.AppendLine("### StdOut");
+                    reportBuilder.AppendLine(TruncateTheStart(test.Output.StdOut, 50_000));
```

Comment:
could it be that the meat is ever at the end? ie would be better to take the first 50_000, then the last (length - 50_000 > 0 ? 100 : 0) or something

idk

Reply:
My thinking was that we might see failed resources or exceptions towards the end. And the beginning would be more of starting up messages. Though the end also last few lines of shutdown messages.

Reply:
I don't have a strong opinion on this though.

Reply:
I mistyped, I meant what about taking the first 25_000, the last 25_000, and removing any overlap. So all you lose is in the middle

Reply:
Done.

Reply:
This is what it will look like now:
```
| [2025-05-15T16:36:23] Aspire.Hosting.Tests.Resources.backend Information: 29: 2025-05-15T16:36:23.8940000Z 14da29b7e816: Download complete
| [2025-05-15T16:36:23] Aspire.Hosting.Tests.Resources.backend Information: 30: 2025-05-15T16

... (snip) ...

... sources.yarp Information: 50: 2025-05-15T16:36:28.1672245Z dbug: Microsoft.Extensions.ServiceDiscovery.ServiceEndpointWatcher[1]
| [2025-05-15T16:36:28] Aspire.Hosting.Tests.Resources.yarp Information: 51: 2025-05-15T16:36:28.1672374Z       Resolving endpoints for service 'http://backend'.
```